src/templates/fr/2019/base.html

```diff
@@ -163,6 +163,7 @@
   "designers": "Design",
   "developers": "Développement",
   "editors": "Édition",
+  "leads": "Chef",
```

Comment:
In French, "Chef" means "Male lead".
I'd rather use "Gestion de projet" (aka. Project Management) to be more inclusive (no need to apply a gender to the job title).

Reply:
FYI: this transformation of job titles into thematics is already done for the other jobs.

Reply:
Fixed! Thanks.